x-pack/osquerybeat/internal/action/action.go

```diff
@@ -37,6 +42,13 @@ func FromMap(m map[string]interface{}) (a Action, err error) {
 		}
 	}
 
+	if v, ok := m["timeout"]; ok {
+		timeout, err = convertToInt64(v)
+		if err != nil {
+			return a, fmt.Errorf("invalid timeout value %v: %w", v, err)
+		}
+	}
```

Comment:
Consider moving this down below the `Action` decl so that the assignment is more local.

Reply:
Moved